controllers/discovery/searchUser.js

```diff
@@ -41,7 +41,8 @@ const SearchUser = async (req, res) => {
                 AND "User"."is_banned" = false) 
             GROUP BY 
                 "User"."user_id"
-            ORDER BY 
+            ORDER BY  
+                "karma_score" DESC,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query is not valid because you cannot order by `karma_score` and then `user_id_follower`. The comma after `karma_score` DESC should be removed. Here's the corrected version:
```diff
-                "karma_score" DESC,
+                "karma_score" DESC
                 "user_id_follower" ASC,
                 "followersCount" DESC
```
Also, ensure that `karma_score` is a column in your "User" table and it has been included in the SELECT statement of your SQL query. If it's not present, this will cause an error.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
mas @fajarism ini untuk yang search user